src/aat/resources/features/F-064/F-064.feature

```diff
@@ -0,0 +1,54 @@
+@F-064
+Feature: F-064: Retrieve workbasket input details for dynamic display
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-217
+  Scenario: should retrieve trigger when the case and event exists
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And it is submitted to call the [retrieve workbasket input details for dynamic display] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [code is HTTP-200 OK]
+    And the response has all other details as expected
+
+  @S-218
+  Scenario: must return a negative response when the request does not provide valid authentication credentials
+    Given a user with [an inactive profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide valid authentication credentials]
+    And it is submitted to call the [retrieve workbasket input details for dynamic display] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [code is HTTP-403 Forbidden]
+    And the response has all other details as expected
+
+  @S-219
```

Comment:
Scenario mismatch with excel:

S-219 | CCD Data Store Api :: Retrieve   workbasket input details for dynamic display |   | should retrieve search inputs
-- | -- | -- | --